src/vivarium_gates_lsff/components/observers.py

```diff
@@ -1,17 +1,204 @@
-from collections import Counter
+import pandas as pd
```

Comment:
Minor, but pandas should be in the 3rd party import block, right? https://www.python.org/dev/peps/pep-0008/#imports